### PR TITLE
Update zwave to use new state values

### PR DESCRIPTION
**Pull request by emlove**
Frontend PR for https://github.com/home-assistant/home-assistant/pull/10413

Screenshot of new zwave card

### CODE REVIEW

**Review by andrey-git — APPROVED** [review]

**Comment by armills** [comment]
Added a potential solution for displaying query stage on the card as well.

File: src/state-summary/state-card-display.html [hunk 1]
```diff
@@ -40,7 +40,15 @@
   }
 
   computeStateDisplay(stateObj) {
-    return window.hassUtil.computeStateState(stateObj);
+    switch (window.hassUtil.computeDomain(stateObj)) {
+      case 'zwave':
```
Comment (andrey-git) [inline src/state-summary/state-card-display.html]: Move this to window.hassUtil.computeStateState

Comment (andrey-git) [reply src/state-summary/state-card-display.html]: Also include stage for "Sleeping", so everything except "ready" will have stage

Comment (armills) [reply src/state-summary/state-card-display.html]: The way we have it configured in the backend right now is that Sleeping will only be sent after the query stage is Complete.

**Pull request merged**